widget/embedded/src/types/event.ts

```diff
@@ -0,0 +1,56 @@
+import type { SelectedQuote } from './quote';
+import type { Wallet } from './wallets';
+
+type Account = Wallet;
+
+export enum QuoteEventTypes {
+  QUOTE_INPUT_UPDATE = 'quoteInputUpdate',
+  QUOTE_UPDATE = 'quoteUpdate',
+}
+
+export enum WalletEventTypes {
+  CONNECTED = 'connected',
+  DISCONNECTED = 'disconnected',
```

Comment:
I suggest changing these to `connect` and `disconnect` to maintain a consistent naming convention with other event types.

Reply:
Considering that the event is emitted after a wallet is connected or disconnected, I think it is better to keep it this way.